charts/lakefs/README.md

```diff
@@ -51,6 +51,26 @@ Sensitive information like `database_connection_string` is given through "secret
 You should give your Kubernetes nodes access to all S3 buckets you intend to use lakeFS with.
 If you can't provide such access, lakeFS can be configured to use an AWS key-pair to authenticate (part of the `lakefsConfig` YAML below).
 
+## Major Chart Upgrades
```

Comment:
`Major` usually means the leftest digit: `Major.Minor.Patch.Build`. Use `Minor` (if you feel the importance is not emphasized enough with that word use an unsaved word like `Major`)

Reply:
I think this change would justify a major version change.

Reply:
I also thought of going with a Major version change, but it feels unaligned with the fact lakeFS stays with version 0.x.x
I think I will change the rephrasing from `Major` to `Notable`